#!/usr/bin/env python
PACKAGE = "hector_thermal_image_conversion"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("min_value",    int_t,    0, "Minimum pixel value for value mapping", -40,  20000, 30000)
gen.add("max_value", int_t, 0, "Maximum pixel value for value mapping",    -40, 20000,   30000)
gen.add("use_raw_threshold", bool_t, 0, "Use raw pixel values instead of degree celsius values", True)

exit(gen.generate(PACKAGE, "hector_thermal_image_conversion", "ThermalImageConversion"))
